staging/README.md

```diff
@@ -1,12 +1,40 @@
+# External Repository Staging Area
+
 This directory is the staging area for packages that have been split to their
 own repository. The content here will be periodically published to respective
 top-level k8s.io repositories.
 
-The code in the `staging/` directory is authoritative, i.e. the only copy of
-the code. You can directly modify such code.
+Repositories currently staged here:
+
+- [`k8s.io/apiextensions-apiserver`](https://github.com/kubernetes/apiextensions-apiserver)
+- [`k8s.io/api`](https://github.com/kubernetes/api)
+- [`k8s.io/apimachinery`](https://github.com/kubernetes/apimachinery)
+- [`k8s.io/apiserver`](https://github.com/kubernetes/apiserver)
+- [`k8s.io/client-go`](https://github.com/kubernetes/client-go)
+- [`k8s.io/kube-aggregator`](https://github.com/kubernetes/kube-aggregator)
+- [`k8s.io/kube-gen`](https://github.com/kubernetes/kube-gen)
```

Comment:
This one is not published yet, so how about [`k8s.io/kube-gen`](https://github.com/kubernetes/kube-gen)(to be published)